src/store/types/memberTypes.ts

```diff
@@ -0,0 +1,51 @@
+// action types
+export const LOAD_MEMBERS = 'LOAD_MEMBERS';
+export const ADD_MEMBER = 'ADD_MEMBER';
+export const DELETE_MEMBERS = 'DELETE_MEMBERS';
+export const LOAD_USER = 'LOAD_USER';
+
+// action payloads
+export interface MemberStateData {
+  id: number;
+  firstName: string;
+  lastName: string;
+  email: string;
+  ChurchId?: number;
```

Comment:
C -> c
```suggestion
  churchId?: number;
```

Reply:
@hchiam if I lowercase that the database won't associate.. soo.. either we change the foreignKey or that's how it's gonna be